pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "k8s.io/kubernetes/pkg/api/unversioned"
+
+type KubeProxyConfiguration struct {
+	unversioned.TypeMeta
+
+	// bindAddress is the IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)
+	BindAddress string `json:"bindAddress"`
+	// cleanupIPTables
+	CleanupIPTables bool `json:"cleanupIPTables"`
+	// healthzBindAddress is the IP address for the health check server to serve on, defaulting to 127.0.0.1 (set to 0.0.0.0 for all interfaces)
+	HealthzBindAddress string `json:"healthzBindAddress"`
```

Comment:
Even though the bindAddress and bindPort are logically separate, its easier for users to describe them in a single field.  It's probably more natural for them to do `127.0.0.1:8443` and this config is for them not for us.